### PR TITLE
[WIP] TELCODOCS-285: Release note added for Topology Aware Lifecycle Operator

### PR DESCRIPTION
**Pull request by amolnar-gh**
Jira issue: https://issues.redhat.com/browse/TELCODOCS-285

Preview: https://deploy-preview-40388--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-topology-aware-lifecycle-operator

For release(s): 4.10

**Moved to a z-stream, do not merge for GA**

### CODE REVIEW

**Comment by amolnar-rh** [comment]
@irinamihai @nishant-parekh Hi! Could you please take a look at the release note?
Additional question, do we know the support level of the feature yet? Thank you!

**Comment by amolnar-rh** [comment]
@irinamihai @nishant-parekh Could you please take a look at the release note?
Additional question, do we know the support level of the feature yet? Thank you!

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -301,6 +301,11 @@ In {product-title} 4.7, _Cluster Logging_ became _Red Hat OpenShift Logging_. Fo
 [id="ocp-4-10-scale"]
 === Scale
 
+[id="ocp-4-10-topology-aware-lifecycle-operator"]
+==== Updating managed clusters with Topology-Aware Lifecycle Operator (Technology Preview)
+
+You can now use the Topology-Aware Lifecycle Operator to perform updates on multiple target clusters by using {rh-rhacm-first} policies. From {product-title} {product-version}, the Operator is available in the OperatorHub. For more information, see xref:../scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc[Topology-Aware Lifecycle Operator].
```
Comment (irinamihai) [inline release_notes/ocp-4-10-release-notes.adoc:307]:
```suggestion
You can now use the Topology-Aware Lifecycle Operator to perform updates on multiple Single Node Openshift target clusters by using {rh-rhacm-first} policies. From {product-title} {product-version}, TALO is available in the OperatorHub. For more information, see xref:../scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc[Topology-Aware Lifecycle Operator].
```

**Comment by amolnar-rh** [comment]
Closing PR. The [TELCODOCS-285](https://issues.redhat.com/browse/TELCODOCS-285) reviews should continue in the consolidated ZTP release notes PR: https://github.com/openshift/openshift-docs/pull/44003

**Pull request closed without merging**